docker-compose-dev.yml

```diff
@@ -7,7 +7,7 @@ services:
       context: .
       dockerfile: Dockerfile
     ports:
-      - "8001:8000"
+      - "8000:8000"
```

Comment:
Is this used on staging, or just for local development?

Reply:
That is just for local development. The swagger ui doesn't work otherwise.